internal/pkg/errorconcept/db.go

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package errorconcept
+
+import (
+	"net/http"
+
+	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
+)
+
+var Database databaseErrorConcept
+
+// DatabaseErrorConcept represents a collection of database error concepts
+type databaseErrorConcept struct {
+	NotFound      dbNotFound
+	NotUnique     dbNotUnique
+	InvalidObject dbInvalidObjectId
```

Comment:
Naming should be consistent; prior change of dbInvalidObject -> dbInvalidObjectId should affect InvalidObject naming.

InvalidObject -> InvalidObjectId